run/system-package/main.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, make_response, request
+import os
+import subprocess
+import sys
+
+app = Flask(__name__)
```

Comment:
nit: I believe python style is usually 2 newlines between blocks like this, or at least after the statement (1 on each side here).

https://github.com/GoogleCloudPlatform/python-docs-samples/search?p=1&q=app+%3D+Flask%28__name__%29&type=&utf8=%E2%9C%93